docs/design-docs/01 - mvp/01-mvp-design.es.md

```diff
@@ -1,11 +1,20 @@
 ## Android Dev Perú App
 
-### Introducción
-Este proyecto involucra las últimas tecnologías relacionadas con el desarrollo de apps Android y Kotlin. Es decir, aquí encontrarás funciones desarrolladas con Jetpack Compose, Kotlin Multiplatform, Gradle, utilizando una arquitectura modular y en capas, testing y todas las habilidades MAD (desarrollo moderno de Android) relacionadas.
```

Comment:
I moved this to the README file. We can safely assume that someone that is reading this design doc has already read the README, and has at least some context about the app.

Reply:
Agree!